src/_includes/flip.html

```diff
@@ -1,18 +1,12 @@
 <div class="flip">
-	<div class="flip-text">Thank You<br />for Shopping<br />With us!</div>
+	<div class="flip-text">Thank you</div>
+	<div class="flip-text italic">For shopping</div>
+	<div class="flip-text">With us</div>
```

Comment:
I think you can put this in a single `div` and wrap "for Shopping" in an `em`.

Also, take a look at the capitalization: "for Shopping" instead of "For shopping".

Reply:
Upon re-reading the issue: https://github.com/undercasetype/fraunces-minisite/issues/46 If we keep the `Style` column, there's no need to have a mix of roman/italic in the demo text.